corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js

```diff
@@ -227,6 +227,8 @@ hqDefine('app_manager/js/forms/advanced/case_config_ui', function () {
                     $('.hq-help-template').each(function () {
                         hqImport("hqwebapp/js/main").transformHelpTemplate($(this), true);
                     });
+
+                    $("#case-load-accordion").sortable("option", "handle", ".panel-heading-case-action");
```

Comment:
This was the best place I could find to add this. I attempted to put it in `initAccordion` https://github.com/dimagi/commcare-hq/blob/acd2fa90cfc6d11147df5de2f52b12eeb2d3363e/corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js#L201 but I don't think the knockout binding was applied yet, even though it is called before https://github.com/dimagi/commcare-hq/blob/acd2fa90cfc6d11147df5de2f52b12eeb2d3363e/corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js#L211

Reply:
Why do you need to do this at all? The `sortableList` binding calls `.sortable(...)`, why doesn't that work on its own?